modin/pandas/test/test_groupby.py

```diff
@@ -3164,14 +3164,6 @@ def test_groupby_agg_provided_callable_warning():
             pandas_groupby.agg(func)
 
 
-def _apply_transform(df):
```

Comment:
dask hangs if I try to submit a function from the global testing scope, so rewritten this function as a lambda